ipe_process_orchestrator/assign_competencies.py

```diff
@@ -129,6 +131,7 @@ def start_assigning_process(self) -> None:
       start_time = time.perf_counter()
       student_grades: List[Dict[str, Any]] = self.get_student_list_with_course_grades()
       if(len(student_grades) == 0):
+        logger.info('0 students received competencies.')
```

Comment:
I simplified the if statement on L57 and added here

Reply:
great!

Reply:
looks good.